test/Inputs/comment_to_something_conversion.swift

```diff
@@ -94,15 +94,15 @@ enum A012_AttachToEntities {
   func f1() {}
 // CHECK: {{.*}}DocCommentAsXML=[<Function file="{{.*}}" line="{{.*}}" column="{{.*}}"><Name>f1()</Name><USR>s:FC14swift_ide_test5Brief2f1FT_T_</USR><Declaration>func f1()</Declaration><Abstract><Para>Aaa.</Para></Abstract><Discussion><Para>Bbb.</Para></Discussion></Function>]
 
-  ///Aaa.
+  /// Aaa.
   ///
-  ///> Bbb.
+  /// Bbb.
```

Comment:
I belive these are test comments for the parser. Changes in this file should be reverted.